added hardik info

hardik-pratap-singh opened the pull request:
I have added all the 3 tasks as was suggested and informed by you.
Kindly merge my pr.

thanks

fineanmol commented:
@hardik-pratap-singh  Please Star ⭐️ the repo to earn "**_hacktober-accepted_**" label for the event.

hardik-pratap-singh commented:
I already loved and shared your repo with my friends ...


Now added star too...

Kindly add the label !
Thanks & Regards
Hardik




On Sat, 1 Oct, 2022, 1:02 pm ANMOL AGARWAL, ***@***.***>
wrote:

> @hardik-pratap-singh <https://github.com/hardik-pratap-singh> Please Star
> ⭐️ the repo to earn "*hacktober-accepted*" label for the event.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/fineanmol/hacktoberfest/pull/34#issuecomment-1264277261>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXENSLKHGTIMPGTWJK5H7EDWA7SJHANCNFSM6AAAAAAQ2JHQZQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

hardik-pratap-singh commented:
> I already loved and shared your repo with my friends ... Now added star too... Kindly add the label ! Thanks & Regards Hardik
> […](#)
> On Sat, 1 Oct, 2022, 1:02 pm ANMOL AGARWAL, ***@***.***> wrote: @hardik-pratap-singh <https://github.com/hardik-pratap-singh> Please Star ⭐️ the repo to earn "*hacktober-accepted*" label for the event. — Reply to this email directly, view it on GitHub <[#34 (comment)](https://github.com/fineanmol/hacktoberfest/pull/34#issuecomment-1264277261)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AXENSLKHGTIMPGTWJK5H7EDWA7SJHANCNFSM6AAAAAAQ2JHQZQ> . You are receiving this because you were mentioned.Message ID: ***@***.***>



> I already loved and shared your repo with my friends ... Now added star too... Kindly add the label ! Thanks & Regards Hardik
> […](#)
> On Sat, 1 Oct, 2022, 1:02 pm ANMOL AGARWAL, ***@***.***> wrote: @hardik-pratap-singh <https://github.com/hardik-pratap-singh> Please Star ⭐️ the repo to earn "*hacktober-accepted*" label for the event. — Reply to this email directly, view it on GitHub <[#34 (comment)](https://github.com/fineanmol/hacktoberfest/pull/34#issuecomment-1264277261)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AXENSLKHGTIMPGTWJK5H7EDWA7SJHANCNFSM6AAAAAAQ2JHQZQ> . You are receiving this because you were mentioned.Message ID: ***@***.***>

## Dear @fineanmol, you have given the tag of hacktoberfest accepted but your repo is showing excluded on the hacktoberfest page .
Why is it so ?
### Kindly look into this please !
